src/ResourceManagement/PowerBIEmbedded/Microsoft.Azure.Management.PowerBIEmbedded/project.json

```diff
@@ -0,0 +1,57 @@
+{
+    "version": "1.0.5-preview",
+    "description": "Microsoft Azure Management Power BI Embedded Library",
+    "authors": [
+        "Microsoft"
+    ],
+    "packOptions": {
+        "tags": [
+            "Microsoft Azure Power BI Embedded management",
+            "Power BI Embedded",
+            "Power BI Embedded management",
+            "REST HTTP client",
+            "windowsazureofficial"
+        ],
+        "projectUrl": "https://github.com/Azure/azure-sdk-for-net",
+        "licenseUrl": "https://raw.githubusercontent.com/Microsoft/dotnet/master/LICENSE"
+    },
+    "buildOptions": {
+        "nowarn": [
+            "CS1591"
+        ],
+        "xmlDoc": true,
+        "delaySign": true,
+        "publicSign": false,
+        "keyFile": "../../../../tools/MSSharedLibKey.snk"
+    },
+
+    "dependencies": {
+        "Microsoft.Rest.ClientRuntime.Azure": "[4.0.2,5.0.0)"
+    },
+
+    "frameworks": {
+        "net45": {
+        },
```

Comment:
See the Compute project as an example.  We are using ClientRuntime.Azure >= 3.1.0 for net 45 and >= 3.3.0 for the netstandard targets